modules/programs/hexchat.nix

```diff
@@ -0,0 +1,323 @@
+{ config, pkgs, lib, ... }:
+
+with builtins;
+with lib;
+let
+  cfg = config.programs.hexchat;
+
+  # Submodules
+  channelOptions = with types;
+    submodule {
+      options = {
+        autoconnect = mkOption {
+          type = nullOr bool;
+          description = "Autoconnect to network";
+          default = false;
+        };
+
+        connectToSelectedServerOnly = mkOption {
+          type = nullOr bool;
+          description = "Connect to selected server only";
+          default = true;
+        };
+
+        bypassProxy = mkOption {
+          type = nullOr bool;
+          description = "Bypass proxy";
```

Comment:
```suggestion
          description = "Bypass proxy.";
```

Reply:
Also, why is this true by default? Is is true by default in hexchat?

Reply:
@superherointj :point_up_2:.